util/perl/OpenSSL/Test.pm

```diff
@@ -529,6 +533,57 @@ END {
     }
 }
 
+=item B<run_background CODEREF>
+
+CODEREF is expected to be the value return by C<cmd> or any of its
+derivatives, anything else will most likely cause an error unless you
+know what you're doing.
+
+C<run_background> spawns the the command returned by CODEREF and returns its
+PID. Unlike C<run>, it currently supports no options. Any output the command
+writes to stdout or stderr is piped to /dev/null.
+
+You should pass the returned PID to C<kill_background> once you are done with
+the spawned process.
+
+=back
+
+=cut
+
+sub run_background {
+    my ($cmd, $display_cmd) = shift->(0);
+    my %opts = @_;
+
+    return () if !$cmd;
+
+    $ENV{HARNESS_OSSL_LEVEL} = $level + 1;
+
+    my $pipe;
+    my $pid = open($pipe, '-|', "exec setsid $cmd >/dev/null 2>/dev/null")
```

Comment:
This is *very* Unix focused...  that's unfortunate.  This will require a bit of hackery to get to work properly on other platforms.  Not necessarily something to do immediately, but it might be wise to add checks if `$^O =~ /^(VMS|MSWin32)$/` and fail this function in that case...